cdk/bin/cdk.ts

```diff
@@ -6,8 +6,17 @@ const app = new App();
 new ManageFrontend(app, 'ManageFrontend-CODE', {
 	stack: 'support',
 	stage: 'CODE',
+	scaling: {
+		minimumInstances: 1,
+	},
+	domain: 'manage.code.theguardian.com.origin.membership.guardianapis.com',
 });
+
 new ManageFrontend(app, 'ManageFrontend-PROD', {
 	stack: 'support',
 	stage: 'PROD',
+	scaling: {
+		minimumInstances: 3,
+	},
+	domain: 'manage.theguardian.com.origin.membership.guardianapis.com',
```

Comment:
Interesting. Could you explain why the different hosted zone is preferred? I don't necessarily think CDK makes the change easier, as we'd still need to co-ordinate changes with any upstream Fastly service. That said, it doesn't make it any more difficult...

Whilst we're discussing Route53, I'd be [keen to understand](https://groups.google.com/a/guardian.co.uk/g/engineering/c/Y6As7_7ovl8/m/jscQnVPZAwAJ) if Route53 is used for reasons other than to remove (old) bottlenecks?

> It's kinda related to my point a few weeks ago, that the original CFN is not necessarily optimal.

Is it worth annotating the existing YAML with some inline comments here?

Reply:
There was an old convention in aws-membership whereby services used the following scheme for their DNS:

<domain-on-code-dot-dev-theguardian-dot-com>.origin.<hostedzone>
<domain-on-theguardian-dot-com>.origin.<hostedzone>

And then either CloudFront or an ELB was used to front them. When we moved most of the stuff to being behind Fastly, this and indeed any convention kinda stopped being enforced, so many apps moved to something more akin to:

<app>[/if stage != PROD]-<stage>[/endif]-backend.<hostedzone>

For further context:

The aws-membership account originally served the membership.theguardian.com stack and had the `membership.guardianapis.com` Hosted Zone , then all the infra from a since-deleted account which served subscribe.theguardian.com was imported into it, adding `subscribe.guardianapis.com` HZ. Then we built contribute.theguardian.com which introduced `contribute.guardianapis.com` HZ, then we built support.theguardian.com which introduced `support.guardianapis.com` HZ. Obviously that's quite messy, and so there's a loose agenda now to ensure everything new goes under the `support.guardianapis.com` HZ. As manage-frontend is tagged with the `support` stack, it should ideally be using the `support.guardianapis.com` HZ and not `membership.guardianapis.com`.

Reply:
We will tackle the namespacing in a later pr